audio/whisper/Portfile

```
@@ -0,0 +1,105 @@
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup   github  1.0
+
+name                whisper
+categories          audio
+github.setup        ggerganov whisper.cpp v1.6.2
+version             1.6.2
+platforms           darwin
+license             MIT
+maintainers         ijams.me:nate
```

A port is always additionally marked `openmaintainer` unless there is an explicit reason why you and only you should be allowed to modify/update the port.  Also you should include your Github handle:

```suggestion
maintainers         {ijams.me:nate @exprez135} \
                    openmaintainer
```